src/main/java/server/response/ResponseBuilder.java

```diff
@@ -1,50 +1,58 @@
 package server.response;
 
-import app.App;
 import server.request.Request;
 import app.MethodApplier;
 import server.response.types.NotFoundResponse;
 import server.response.types.OKResponse;
 
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 
 public class ResponseBuilder {
-    public static Response buildResponse(Request request, App app) {
-        Map<String, MethodApplier> route = app.routes().get(request.path());
+    private Request request;
+    private Map<String, MethodApplier> route;
 
+    public ResponseBuilder(Request request, Map<String, MethodApplier> route) {
+        this.request = request;
+        this.route = route;
+    }
+
+    public Response buildResponse() {
```

Comment:
I'm wondering if other developers might expect `buildResponse` to just be `build`.